doc/fpmsyncd/frr_sonic_protobuf_channel.md

```diff
@@ -109,40 +156,29 @@ To implement these modifications, a PR has been submitted to the FRR repository
 
 ### 7.2. SONiC fpmsyncd changes
 
-On the SONiC side, SONiC's `fpmsyncd` is responsible for collecting the FIB state generated by FRR and dumping its content into the APPL_DB. Upon initialization, SONiC's `fpmsyncd` waits for incoming FPM connections on the FPM port (TCP port 2620). Subsequently, it enters an indefinite loop to receive FPM messages. Each FPM message is then routed to `FpmLink`, which is responsible for parsing the message and passing it to the appropriate handler (`RouteSync`). Currently, `RouteSync` is only capable of interpreting messages containing routes encoded in Netlink format.
-
-We perform the following changes:
-
-* We extend SONiC's `fpmsyncd` to accept connections from multiple FPM clients. This is required to support receiving information from both FPM Netlink (`dplane_fpm_nl`) and FPM Protobuf (`dplane_fpm_pb`) modules at the same time.
-* We extend SONiC's `fpmsyncd` to parse messages encoded in protobuf.
-* We extend SONiC's `RouteSync` by adding an handler to parse information encoded in Protobuf format, such as SRv6 routes and SIDs.
+We make some changes in some of the SONiC components, namely `FpmLink` and `RouteSync`, which are part of the `fpmsyncd` process. The reason behind these changes is to enable `fpmsyncd` to process Protobuf messages received from FRR and write the necessary information (such as routes and SRv6 SIDs) to the SONiC APPL DB.
```

Comment:
Today, in `fpmsyncd` registers Message handler for `RTM_NEWROUTE` and `RTM_DELROUTE for route messages. Will there be similar mechanism to register message handler for protobuf message types?

Reply:
Ditto. In the updated HLD, we are no longer using Protobuf.